pyproject.toml

```diff
@@ -21,8 +21,8 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
     "click>=7.0",
-    "rasterio>=1.3.3",
-    "morecantile>=5.0,<6.0",
+    "rasterio>=1.4.1",
+    "morecantile>=6.0,<7.0",
```

Comment:
I think we can still use 5.0 so the update should be 

```
"morecantile>=5.0,<7.0",
```

as we did in rio-tiler

Reply:
the `breaking` change in morecantile are https://github.com/developmentseed/morecantile/blob/main/CHANGES.md#600-2024-10-16 which are not used in rio-cogeo